app/MMB/src/scenes/timeline/Timeline.js

```diff
@@ -0,0 +1,65 @@
+// @flow
+
+import * as React from 'react';
+import { ScrollView } from 'react-native';
+import {
+  type RouteNamesType,
+  type NavigationType,
+} from '@kiwicom/mobile-navigation';
+import { DateUtils } from '@kiwicom/mobile-localization';
+
+import Events from './Events';
+
+type Props = {|
+  +navigation: NavigationType,
+|};
+
+export default class Timeline extends React.Component<Props> {
+  navigate = (key: RouteNamesType) => {
+    this.props.navigation.navigate({
+      routeName: key,
+      key: `key-${key}`,
+    });
+  };
+
+  render = () => {
+    const events = [
```

Comment:
I think this is a bad way to build applications. Shouldn't you create the API first? You made up this data structure but you are already throwing your work away because this is not going to be the same in API...